CHANGELOG.md

```diff
@@ -7,7 +7,10 @@ Please format the changes as follows:
 + BugFixes:
 + Updates:
 
+# 1.0.26
+
 # 1.0.25
+This version did not get published due to issues encountered in App Service. Mitigation was taken by repackaging the previously released version (1.0.21) as 1.0.25.
```

Comment:
nit: space before "The"

Reply:
I'm not quite sure what you mean, the spacing looks fine to me?

Reply:
Huh, that's odd. When on the conversation tab, it shows that it has a space, but when on the file changes tab, it does not.